compiler/ballerina-lang/src/main/java/org/wso2/ballerinalang/compiler/semantics/analyzer/TypeChecker.java

```diff
@@ -1310,11 +1310,10 @@ private boolean validateTableKeyValue(List<String> keySpecifierFieldNames,
                 dlog.error(recordLiteral.pos,
                         DiagnosticErrorCode.KEY_SPECIFIER_FIELD_VALUE_MUST_BE_CONSTANT_EXPR, fieldName);
                 resultType = symTable.semanticError;
```

Comment:
Isnt this order dependent now on the order of the `recordLiteral` from `recordLiterals`?
If the last member resolves to `symTable.semanticError` we have existing behaviour.

Reply:
I will create an issue for setting resultType here in this function and not related to this PR.

Reply:
Created an issue for this
https://github.com/ballerina-platform/ballerina-lang/issues/36774